libultraship/libultraship/Archive.h

```diff
@@ -37,11 +37,13 @@ namespace Ship
 		std::vector<SFILE_FIND_DATA> ListFiles(const std::string& searchMask) const;
 		bool HasFile(const std::string& searchMask) const;
 		const std::string* HashToString(uint64_t hash) const;
+        std::vector<uint32_t> gameVersions;
 	protected:
 		bool Load(bool enableWriting, bool genCRCMap);
 		bool Unload();
 	private:
 		std::string MainPath;
+        std::string BasePath;
```

Comment:
I'm not sure that we need a basePath. The other archives can just be loaded as a patch.